cmd/cluster-agent/custommetrics/server.go

```diff
@@ -125,9 +126,9 @@ func (o *DatadogMetricsAdapter) Config() (*apiserver.Config, error) {
 	}, nil
 }
 
-// StopServer closes the connection and the server
+// clearServerResources closes the connection and the server
 // stops listening to new commands.
-func StopServer() {
+func clearServerResources() {
```

Comment:
Not sure why this was renamed isn't `stop()` enough?

Reply:
the stop chan is used for the cluster checks as well, so closing it leads to stopping the servers, but the actual function just propagates the signal, we don't stop the server per se.